src/useradd.c

```diff
@@ -589,6 +589,17 @@ set_defaults(void)
 		goto err_free_def;
 	}
 
+	/*
+	 * Change the permissions of the new file to 644
+	 */
+	if (chmod(new_file, 0644) != 0) {
+		fprintf (stderr,
```

Comment:
Please don't add a whitespace here.  (yes, some old code has it; please ignore that old code).